src/bolos/cx_ec.c

```diff
@@ -868,6 +868,81 @@ static cx_curve_weierstrass_t const C_cx_BrainpoolP512t1 = {
 
 };
 
+/* ------------------------------------------------------------------------ */
+/* ---                         Stark256                                 --- */
+/* ------------------------------------------------------------------------ */
+
+static uint8_t const C_cx_Stark256_a[] = {
```

Comment:
The curve parameters are also defined in `cx_ec_domain.c`. Is it possible to define them once and declare them as `extern` in a `.h` file?

Reply:
The curve parameters are actually useless in this file as we are not planning on adding the support of Stark on older SDK, removing them :)